### PR TITLE
Single touch tags

### PR DESCRIPTION
**Pull request by gonzalovelasco**
Screenshot 2023-09-05 at 11 29 18

<br> 

Screenshot 2023-09-05 at 11 29 24

### CODE REVIEW

File: lib/screens/note/components/ContentField/textfield_toolbar.dart [hunk 1]
```diff
@@ -177,6 +175,34 @@ class TextFieldToolbar extends ConsumerWidget implements PreferredSizeWidget {
...
+    switch (toolbarState) {
+      case ToolbarState.edit:
+        return editButtons;
+      case ToolbarState.markdown:
+        return mdButtons;
+      case ToolbarState.tags:
+        return tags?.map((tag) {
+              return KeyboardButton(
+                  child: Text(
+                    tag,
```
Comment (matthewwong525) [inline lib/screens/note/components/ContentField/textfield_toolbar.dart:187]:
```suggestion
                    '#$tag',
```

Comment (matthewwong525) [reply lib/screens/note/components/ContentField/textfield_toolbar.dart:187]: Also, do you watch anime?

Comment (gonzalovelasco) [reply lib/screens/note/components/ContentField/textfield_toolbar.dart:187]: yes, sometimes.

Comment (matthewwong525) [reply lib/screens/note/components/ContentField/textfield_toolbar.dart]: me too! good to hear that :D

File: lib/screens/note/components/ContentField/textfield_toolbar.dart [hunk 2]
```diff
@@ -177,6 +175,34 @@ class TextFieldToolbar extends ConsumerWidget implements PreferredSizeWidget {
...
+        return editButtons;
+      case ToolbarState.markdown:
+        return mdButtons;
+      case ToolbarState.tags:
+        return tags?.map((tag) {
+              return KeyboardButton(
+                  child: Text(
+                    '#$tag',
+                    style: const TextStyle(
+                      fontSize: 20,
```
Comment (matthewwong525) [inline lib/screens/note/components/ContentField/textfield_toolbar.dart:189]:
```suggestion
                      fontSize: 14,
```

**Review by matthewwong525 — APPROVED** [review]
LGTM

**Pull request merged**